tests/conftest.py

```diff
@@ -86,31 +86,15 @@ def cli_runner():
 
 
 @pytest.fixture
-def dummy_agent_tim(test_filesystem, monkeypatch) -> Path:
+def dummy_agent_tim(test_packages_filesystem) -> Path:
     """Fixture for dummy agent tim."""
-
-    monkeypatch.syspath_prepend(test_filesystem)
-    assert Path.cwd() == Path(test_filesystem)
-
+    assert Path.cwd() == Path(test_packages_filesystem)
     agent = PublicId.from_str("dummy_author/tim")
-    command = f"adev create {agent!s} -t eightballer/base "
+    command = f"adev create {agent!s} -t eightballer/base --no-clean-up"
     command_executor = CommandExecutor(command)
     result = command_executor.execute(verbose=True, shell=True)
     if not result:
         msg = f"CLI command execution failed: `{command}`"
         raise ValueError(msg)
-
-    os.chdir(str(Path.cwd() / agent.name))
-
-    commands = (
-        "aea generate-key ethereum",
-        "aea add-key ethereum",
-    )
-    for command in commands:
-        command_executor = CommandExecutor(command.split())
-        result = command_executor.execute(verbose=True)
-        if not result:
-            msg = f"CLI command execution failed: `{command}`"
-            raise ValueError(msg)
-
-    return Path.cwd()
+    os.chdir(agent.name)
+    return True
```

Comment:
why return a useless boolean rather than the path? Also breaks disabled tests and type annotation now incorrect